docs/UserGuide.md

```diff
@@ -63,27 +74,171 @@ Start keeping track of your internships easily with <span style="color: #f66a0a;
+
+💡 **Try It Out:**<br>
+
+1. Input:<br>
+   `clear`<br>
+   ![Before clearing](./images/clear/clear-before.png)<br><br>
+
+2. Successful Output after clearing entries:<br>
+   ![After successfully clearing](./images/clear/clear-after.png)
+   There are no more internships in the list.
+</div>
+
+### Adding an internship entry: `add`
+
+Next, let us try to add an internship entry. 
+
+<div markdown="span" class="alert alert-success">
+
+💡 **Try It Out:**<br>
+
+Key in the following command in the CLI: `add /com Tiktok /desc create new recommendation engine /status ongoing /poc jane yeo /email hr@tiktok.com
+/phone 90890301 /loc remote /role Software Intern`
+
+Successful output after adding the entry:<br>
+   ![After successfully adding](./images/tutorial/add.png)
+   You can now see your new internship with the details you entered in the **Internship List Display!**
+</div>
+
+### Editing an internship entry: `edit`
+
+Oh no! You made a mistake in the email address of the contact person. Let's edit the email address.
+
+<div markdown="span" class="alert alert-success">
+
+💡 **Try It Out:**<br>
+Key in the following command in the CLI: `edit 1 /email janeyeo@gmail.com`
+
+Successful output after editing the entry:<br>
+![After successfully editing](./images/tutorial/edit.png)
+Your internship now has the updated email that you keyed in.
+</div>
 
+### Find internship entries: `find`
+
+Before you continue, add a few more internship entries using these commands:
+1. `add /com Google /desc create new search engine /status to_apply /poc john tan /email johntan@gmail.com /phone 98765432 /loc local /role Software Intern`
+2. `add /com Facebook /desc create new social media platform /status to_apply /poc jane tan /email facebookhr@gmail.com /phone 87654321 /loc remote /role Software Intern`
+
+Successful output after adding the entries:<br>
+![After successfully adding more](./images/tutorial/addmore.png)
+
+Let's say you want to find all internships that you want to apply for, that give you a software intern role.
+`find` is the command you need!
+
+<div markdown="span" class="alert alert-success">
+
+💡 **Try It Out:**<br>
+
+Key in the following command in the CLI: `find WITHALL /status to_apply /role Software Intern`
+
+Successful output after finding all entries that satisfy the conditions:<br>
+![After successfully finding](./images/tutorial/find.png)
+Your **Internship List Display** now only has all the internships satisfying both conditions.
+</div>
+
+### Deleting an internship entry: `delete`
+You decide that to no longer pursue the internship at Facebook. Let's delete that entry.
+
+<div markdown="span" class="alert alert-success">
+
+💡 **Try It Out:**<br>
+
+Key in the following command in the CLI: `delete 2`
+
+Successful output after deleting the entry:<br>
+![After successfully deleting](./images/tutorial/delete.png)
+The Facebook internship entry has been removed, and is no longer present in your database.
+</div>
```

Comment:
Same comment as per what I wrote for the 'add' command

Reply:
same